ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -56,6 +56,9 @@ periodics:
         # also tag the image into the art-latest stream
         oc tag "${to_quay}" "ocp/${BRANCH}-art-latest:${TAG}"
 
+        # also tag the image into the private art-latest stream
+        oc tag "${to_quay}" "ocp-priv/${BRANCH}-art-latest-priv:${TAG}"
```

Comment:
I was under the assumption that we'd have a private RHCOS build that we publish to the private stream, not this?

Reply:
We will. That's going to take some more rhcos pipeline work. With this intermediate step, we can at least handle embargoes that don't impact rhcos.